sources/platform/runtime/machine-image/cli-versions.md

```diff
@@ -83,6 +85,7 @@ Please refer to the [Azure CLI Reference page](https://docs.microsoft.com/en-us/
 | Version  | OS |  Machine Images
 |----------|---------|-------------|
 | 17.10.0-ce | - Windows 16 | [v6.5.4](/platform/runtime/machine-image/ami-v654/)
+| 17.06.2-ee-5 | - Windows 16 |[v6.6.2](/platform/runtime/machine-image/ami-v662/)
```

Comment:
Is this correct? This is supposed to be the version of Docker CLI inside the w16 image, not the docker server version running on the host. If it was 17.10 in 6.5.4, why did we go back to 17.06 in this 6.6.2?